### ISSUE TITLE
SimulatorOptions not recognize CouplingMap object

### ISSUE THREAD

**Issue by MaldoAlberto** [Qiskit/qiskit-ibm-runtime#1071]
**Describe the bug**

When using Option.simulation or SimulatorOptions in coupling_map it does not recognise the CouplingMap object, it only considers a list, otherwise the error occurs. 

`TypeError: Object of type CouplingMap is not JSON serializable`

**Steps to reproduce**
```

from qiskit.providers.fake_provider import FakeManila,FakeManilaV2
from qiskit_aer.noise import NoiseModel
from qiskit_ibm_runtime import QiskitRuntimeService, Session, Sampler, Estimator, Options
from qiskit.test.reference_circuits import ReferenceCircuits

# Initialize account.
service = QiskitRuntimeService()

options = Options(optimization_level=1)
options.execution.shots=20000
bell = ReferenceCircuits.bell()


# Make a noise model
noise_model = NoiseModel.from_backend(FakeManilaV2())

# Set options to include the noise model
options = Options()

from qiskit_ibm_runtime.options import SimulatorOptions

simop = SimulatorOptions(noise_model= noise_model,
basis_gates= FakeManilaV2().operation_names,                        
coupling_map= FakeManilaV2().coupling_map) 

options.simulator = simop



# Set number of shots, optimization_level and resilience_level
options.execution.shots = 20000
options.optimization_level = 0
options.resilience_level = 0

with Session(service=service, backend="ibmq_qasm_simulator") as session:
    # Submit a request to the Sampler primitive within the session.
    sampler = Sampler(session=session, options=options)
    job = sampler.run([bell]*40)
    print(f"Sampler results: {job.result()}")
```
    
**Expected behavior**

works if I put 

`simop = SimulatorOptions(noise_model= noise_model,
basis_gates= FakeManilaV2().operation_names, 
coupling_map= [FakeManilaV2().coupling_map.get_edges()[i] for i in range(len(FakeManilaV2().coupling_map.get_edges()))])`

**Suggested solutions**

add it in   `class RuntimeEncoder(json.JSONEncoder):`  the object CouplingMap from https://github.com/Qiskit/qiskit-ibm-runtime/blob/main/qiskit_ibm_runtime/utils/json.py

or in Options as in SimulatorOptions indicate internal when is FakeV generate a list .

**Additional Information**

- **qiskit-ibm-runtime version**: 0.9.4
- **Python version**: 3.7
- **Operating system**: Ubutnu 21.04

**Comment by merav-aharoni** [comment]
I believe this was fixed in version 0.11.0. Please update your version and check if you still see this bug.

**Comment by MaldoAlberto** [comment]
@merav-aharoni  yeah is working now, thank you so much :D
